### ISSUE TITLE
push to other view controllers dose not work

### ISSUE THREAD

**Issue by ShimaDarya** [malcommac/ScrollingStackContainer#7]

**Comment by azamouchi** [comment]
Yep , that's because only the view of the child ViewControllers is being added to the ScrollingStack,

https://github.com/malcommac/ScrollingStackContainer/blob/593b2890d54d8f6cd01b6235fab811094c74b662/Sources/ScrollingStackContainer/ScrollingStackController.swift#L206

the ViewControllers need to bed added as childViewControllers in ScrollingStackController and then you we can use the following to push other view controllers form any childViewController

`self.parent?.navigationController?.pushViewController(vc, animated: true)`

Will try to make a PR to fix this.
